src/components/ChataTable/ChataTable.js

```diff
@@ -1374,10 +1372,14 @@ export default class ChataTable extends React.Component {
     const languageCode = getDataFormatting(this.props.dataFormatting).languageCode
     const currentRowsFormatted = new Intl.NumberFormat(languageCode, {}).format(currentRowCount)
     const totalRowsFormatted = new Intl.NumberFormat(languageCode, {}).format(totalRowCount)
+    const rowLimit = this.props.response?.data?.data?.row_limit
+    const rowLimitFormatted = new Intl.NumberFormat(languageCode, {}).format(rowLimit)
 
     return (
       <div className='table-row-count'>
-        <span>{`Scrolled ${currentRowsFormatted} / ${totalRowsFormatted} rows`}</span>
+        <span>{`Scrolled ${currentRowsFormatted} / ${
+          totalRowCount > 50000 ? rowLimitFormatted + '+' : totalRowsFormatted
```

Comment:
What does the network response when the data returned > 50000? I think there will be a key `count_sql` that will = 1. I think this would be a better condition to check

Reply:
resolved, used data set value vs hard coded value